rootchain/payload/validator_set_payload.go

```diff
@@ -0,0 +1,55 @@
+package payload
+
+import (
+	"github.com/0xPolygon/polygon-edge/rootchain"
+	rootProto "github.com/0xPolygon/polygon-edge/rootchain/proto"
+	"google.golang.org/protobuf/proto"
+)
+
+// ValidatorSetPayload defines a payload that contains
+// an array of tuples (validatorAddr, BLSPublicKey)
+type ValidatorSetPayload struct {
+	setInfo []ValidatorSetInfo
+}
+
+type ValidatorSetInfo struct {
+	Address      []byte
+	BLSPublicKey []byte
+}
+
+func NewValidatorSetPayload(setInfo []ValidatorSetInfo) *ValidatorSetPayload {
+	return &ValidatorSetPayload{
+		setInfo: setInfo,
+	}
+}
+
+func (v *ValidatorSetPayload) Get() (rootchain.PayloadType, []byte) {
+	// TODO @dbrajovic, do we even need to handle this error?
```

Comment:
I don't think we need to check for this error, as there will only be this particular type (`ValidatorSetPayload`)

Reply:
Removed 👍 

[1b6d795](https://github.com/0xPolygon/polygon-edge/pull/717/commits/1b6d7953e2fa8f5ebb3a2580b89471a52c5f5471)